### PR TITLE
2.3 - UI Updates (My profile)

### PR DESCRIPTION
**Pull request by ngvtuan**
##### Commits
- [x] 2.3_my profile.jpg
- [x] 2.3.1_my profile_edit-1.jpg
- [x] 2.3_my board.jpg
- [x] 2.3_my profile_empty.jpg
- [x] 2.4_security password.jpg
- [x] 2.5_my security.jpg

##### FreedCamp
- [x] [2.3 - UI Updates (My profile)] (https://freedcamp.com/view/2956183/tasks/40590107)

### CODE REVIEW

**Comment by sanjanadhikari1** [comment]
Could you please resolve conflicts first?

File: src/features/profile/components/MyProfileGeneral.tsx [hunk 1]
```diff
@@ -8,22 +11,22 @@ import Select, { TOption } from './Select/Select'
 export type TProfileGeneral = {
   editMode: boolean
   isEmpty: boolean
+  nativeCurrency?: string
```
Comment (CrazyOptimist) [inline src/features/profile/components/MyProfileGeneral.tsx:14]: Could you add a typescript enum for nativeCurrency?

File: src/features/profile/components/MyProfileGeneral.tsx [hunk 2]
```diff
@@ -32,36 +35,47 @@ const ProfileGeneral = ({
...
     setBio(isEmpty ? 'None' : data.bio)
   }, [isEmpty])
 
+  useEffect(() => {
+    const getNativeCurrency = async (): Promise<void> => {
+      if (!nativeCurrency) {
+        return
+      }
+
+      const result = await Convert(15).from('USD').to(nativeCurrency)
```
Comment (CrazyOptimist) [inline src/features/profile/components/MyProfileGeneral.tsx:62]: 15 is for current PSL price in USD as I can see, please move it to constant for now

**Review by CrazyOptimist — CHANGES_REQUESTED** [review]

File: src/features/profile/components/MyProfileGeneral.tsx (hunk 1 above)
Comment (ngvtuan) [reply src/features/profile/components/MyProfileGeneral.tsx]: Fixed

**Review by CrazyOptimist — APPROVED** [review]

**Review by sanjanadhikari1 — APPROVED** [review]

File: src/common/assets/icons/ico-pencil-transparent.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (ngvtuan) [reply src/common/assets/icons/ico-pencil-transparent.svg:1]: @sanjanadhikari1 I have used this one `react-inlinesvg` https://www.npmjs.com/package/react-inlinesvg

File: src/features/profile/components/ChangePastelUsernameModal.tsx [hunk 4]
```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react'
```
[the remark this answers is not in the document]
Comment (ngvtuan) [reply src/features/profile/components/ChangePastelUsernameModal.tsx]: Yes I have used your component (`ChangePastelUsernnameModal`) and I have removed mine

**Pull request merged**